runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -167,4 +167,12 @@
   Boolean isShutdownSourcesOnFinalWatermark();
 
   void setShutdownSourcesOnFinalWatermark(Boolean shutdownOnFinalWatermark);
+
+  @Description(
+      "Interval in milliseconds for sending latency tracking marks from the sources to the sinks. " 
```

Comment:
The build is failing because spotless is complaining about the space after this line.

Reply:
Merged this with a whitespace fix applied in the merge commit.

Reply:
Sorry, I was offline to a while. Thanks a lot for getting this through!